### DISCUSSION TITLE
Use MarkDoc for MD extention

### DISCUSSION THREAD

**Discussion by tomByrer** [nuejs/nue#392] [Ideas]
[MarkDoc](https://markdoc.dev/) is doing the same thing as your "Markdown extensions", but is built by a major corp & has Astro support.

**Comment by tipiirai** [comment]
Yeah. I'm aware of MarkDoc

**Reply by tipiirai** [reply]
The reason I chose not to use MarkDoc is that it is not focused on content enough. You have the possibility to inject programming constructs and HTML inside the content file, which does not fit into the core philosophies of Nue: separation of concerns and progressive enhancement.
